ajna/v2/ethereum/tasks.py

```diff
@@ -57,8 +57,9 @@
 
 @app.task
 def fetch_market_price_task():
+    chain = Ethereum()
     models = EthereumModels()
```

Comment:
If I remember correctly, EthereumModels are attached to Ethereum on root, so in this case, chain and models can be used in the same way, so you could replace `models` with `chain` and you wouldn't need to pass both down to other functions. Just chain should be enough